tests/test_text/test_dispersion.py

```diff
@@ -0,0 +1,53 @@
+# tests.test_text.test_freqdist
+# Tests for the frequency distribution visualization
+#
+# Author:   Rebecca Bilbro
+# Github:   @rebeccabilbro
+# Created:  2017-03-22 15:27
+#
+# Copyright (C) 2018
+# For license information, see LICENSE.txt
+#
+# ID: test_freqdist.py [bd9cbb9] rbilbro@districtdatalabs.com $
+
+"""
+Tests for the frequency distribution text visualization
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import sys
+import pytest
+
+from yellowbrick.text.dispersion import *
+from tests.dataset import DatasetMixin
+from tests.base import VisualTestCase
+
+
+##########################################################################
+## DispersionPlot Tests
+##########################################################################
+
+class FreqDistTests(VisualTestCase, DatasetMixin):
+
+    @pytest.mark.xfail(
+        sys.platform == 'win32', reason="images not close on windows"
+    )
+    def test_integrated_dispersionplot(self):
+        """
+        Assert no errors occur during DispersionPlot integration
+        """
+        corpus = self.load_data('hobbies')
+	
+        docs = ' '.join(corpus.data)
+        text = docs.split()
```

Comment:
Great solution! If you're feeling frisky you could also get it in one step as:

```python
text = [word for doc in corpus.data for word in doc.split()]
```

We might also want to test a generator in a separate test (to ensure both lists and generators work with the visualizer):

```python
from itertools import chain 

text = chain(*map(str.split, corpus.data))
```

Note the `*` before the map; that expands the map into chain arguments. Chain and map are both generators in this case.